src/styles/main.scss

```diff
@@ -0,0 +1,10 @@
+@import "./utils/variables";
+@import "./utils/mixins";
+@import "./utils/extends";
+
+@import  "./blocks/stopwatch";
+
+* {
+  box-sizing: border-box;
+  margin: 0;
```

Comment:
do not reset margins for  all elements
reset for some special elements you need

Reply:
done